src/elements/data-structure.js

```diff
@@ -8,34 +8,6 @@ export default function (namespace) {
         this.content = namespace.toElement(this.content);
       }
     }
-
-    toValue() {
-      return this.content && this.content.toValue();
-    }
-
-    toRefract() {
-      const refract = super.toRefract();
-
-      if (Array.isArray(this.content)) {
-        refract.content = this.content.map(item => item.toRefract());
-      } else {
-        refract.content = this.content.toRefract();
-      }
-
-      return refract;
-    }
-
-    fromRefract(doc) {
-      super.fromRefract(doc);
-
-      if (Array.isArray(doc.content)) {
-        this.content = doc.content.map(item => namespace.fromRefract(item));
-      } else {
-        this.content = namespace.fromRefract(doc.content);
-      }
-
-      return this;
-    }
```

Comment:
Removed because the separate serialiser can handle dataStructure correctly, and also that `toValue` base implementation now supports the behaviour for dataStructure.